.github/workflows/lint_check.yml

```diff
@@ -29,7 +29,7 @@ jobs:
           python -m pip install --upgrade pip
           python -m pip install -e .[dask]
           python -m pip install -e .[spark]
-          python -m pip install -r dev-requirements.txt
-          python -m pip install -r test-requirements.txt
+          python -m pip install -e .[dev]
+          python -m pip install -e .[test]
```

Comment:
The dev requirements include everything, so I don't think you need any of the other install commands.

Reply:
updated